DEPS

```diff
@@ -98,7 +98,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + 'da131f62020132fa0df9008ad5951bb8ac591746',
+  'src': 'https://github.com/flutter/buildroot.git' + '@' + '587ecd678c505e8d614b5f6ea498477c49c09e84',
```

Comment:
won't this need to get changed once the buildroot PR lands?

Reply:
Yes indeed - updated. 👍